c_deps/Makefile

```diff
@@ -0,0 +1,63 @@
+DIR != pwd
+INST_DIR = ${DIR}/inst
+PYTHON=python2.7
+
+PROCESSOR_TRACE_REPO = https://github.com/01org/processor-trace.git
+PROCESSOR_TRACE_V = 0ff8b29b2fd2ebfcc47a747862e948e8b638a020
+PROCESSOR_TRACE_SOURCE = processor-trace
+
+XED_REPO = https://github.com/intelxed/xed
+XED_V = bd73bdb2eefac1896b886e6a39e1794ae5267f7f
+XED_SOURCE = xed
+
+MBUILD_REPO = https://github.com/intelxed/mbuild
+MBUILD_V = bb9123152a330c7fa1ff1a502950dc199c83e177
+MBUILD_SOURCE = mbuild
+
+.PHONY: libipt
+
+all: libipt
+
+libipt: ${INST_DIR}/bin/ptdump
+
+${INST_DIR}:
+	install -d ${INST_DIR}/bin
+
+#
+# Fetch targets
```

Comment:
Can I suggest we remove the `#` lines above and below (i.e. have 1 blank line above and below this line)? I'd actually find it makes the important line pop out better.